### DISCUSSION TITLE
Reusing the same footnote multiple times

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#1799] [Q&A] [answered]
Hi,

I'm trying to reuse a footnote in multiple places in a book chapter like so:
```md

The force in the x direction of the beam [^*]
...
...
...
The torque in the x direction of the beam [^*]
...

[^*]: equal and opposite at each end

```

Two problems - you can't use symbols/letters it seems (#1741) which I would prefer. Ignoring that; when I render the document it creates a new footnote for each time I have used it in the document instead of just having the one expected footnote.

![image](https://user-images.githubusercontent.com/10863809/183993080-8c992f77-ff5e-4cf0-b39b-93731e2b75ae.png)

How do I fix this behaviour and reuse the footnote?

**Comment by jjallaire** [comment, marked as answer]
This is unfortunately a limitation of Pandoc (our markdown rendering engine). You can see the discussion here: https://github.com/jgm/pandoc/issues/1603

**Reply by jpmorr** [reply]
Annoyingly after 8 years it's still not resolved - it can't be that complicated!! Thanks for the answer.
